salt/cloud/clouds/aliyun.py

```diff
@@ -339,7 +339,7 @@ def list_nodes_full(call=None):
         provider = comps[0]
 
     __opts__['update_cachedir'] = True
-    salt.utils.cloud.cache_node_list(ret, provider, __opts__)
+    __utils__['cloud.cache_node_list'](ret, provider, __opts__)
```

Comment:
That's fine. If you want to do a deprecation path, we can do that but let's just go ahead and merge this as it is. Thanks.